src/assets/utils/utils.js

```diff
@@ -0,0 +1,15 @@
+export const getObjValue = (obj, path) => {
```

Comment:
```suggestion
export const getObjValue = (obj, path, defaultValue = false) => {
```
Переменную `defaultValue` можно вынести в аргументы функции и задать значение по умолчанию, иначе его никак не модифицировать будет в дальнейшем.